xclim/sdba/nbutils.py

```diff
@@ -204,7 +217,17 @@ def _escore(tgt, sim, out):
     out[0] = w * (sXY + sXY - sXX - sYY) / 2
 
 
-@njit
+@njit(
+    #    [
```

Comment:
Remove commented code ?
Also, I'm not well versed in numba but my understanding is that it mainly speeds up for loop over numpy arrays.
Does it accelerate anything else ? In particular, what is the purpose of numba for the functions of this module ?

Reply:
Never mind, I think I simply removed the problematic headers in the concerned functions, the question remains on what to do with the rest.

Reply:
Yes, I was exploring other compilation options for the jitting, particularly for testing, to avoid having to do a dry-run before every test, I believe.

It can be axed!